wdl2cwl/main.py

```diff
@@ -87,7 +87,9 @@ def get_command(
     while index < len(command):
 
         # if you have ~{
-        if command[index] == "~" and command[index + 1] == "{":
+        if (command[index] == "~" and command[index + 1] == "{") or (
+            command[index] == "$" and command[index + 1] == "{"
```

Comment:
Only replace `${…}` style if the command block is in the `command { ... }`style (and never if the command block is in the `command <<< >>>` style): https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#expression-placeholders

Also, don't replace if the identifier is not the name of any of the inputs.